addons/web_editor/static/src/js/editor/snippets.editor.js

```diff
@@ -71,31 +291,34 @@ var SnippetEditor = Widget.extend({
         // Initialize move/clone/remove buttons
         if (this.isTargetMovable) {
             this.dropped = false;
-            this.$el.draggable({
-                greedy: true,
-                appendTo: this.$body,
-                cursor: 'move',
-                handle: '.o_move_handle',
-                cursorAt: {
-                    left: 18,
-                    top: 14
-                },
-                helper: () => {
-                    var $clone = this.$el.clone().css({width: '24px', height: '24px', border: 0});
-                    $clone.appendTo(this.$body).removeClass('d-none');
-                    return $clone;
-                },
-                start: this._onDragAndDropStart.bind(this),
-                stop: (...args) => {
-                    // Delay our stop handler so that some summernote handlers
-                    // which occur on mouseup (and are themself delayed) are
-                    // executed first (this prevents the library to crash
-                    // because our stop handler may change the DOM).
-                    setTimeout(() => {
-                        this._onDragAndDropStop(...args);
-                    }, 0);
-                },
-            });
+            this.setDraggable(
+                this.$el,
+                {
```

Comment:
Avoid indenting the old code for no reason (potential code history breaking + review harder, for no reason):
```javascript
this.setDraggable(this.$el, {
    // ...
});
```